webapp/src/components/AssetPage/ItemDetail/ItemDetail.tsx

```diff
@@ -109,7 +132,46 @@ const ItemDetail = ({ item }: Props) => {
       box={null}
       showDetails
       actions={<SaleActionBox asset={item} />}
-      below={<TransactionHistory asset={item} />}
+      below={
+        <div className={styles.tableContainer}>
+          <div className={styles.filtertabsContainer}>
+            <Tabs isFullscreen>
+              <Tabs.Tab
+                active={belowTab === BelowTabs.LISTINGS}
+                onClick={() => setBelowTab(BelowTabs.LISTINGS)}
+              >
+                <div className={styles.tabStyle}>
+                  {t('transaction_history.title')}
+                </div>
+              </Tabs.Tab>
+              <Tabs.Tab
+                active={belowTab === BelowTabs.OWNERS}
+                onClick={() => setBelowTab(BelowTabs.OWNERS)}
+              >
+                {t('owners_table.owners')}
+              </Tabs.Tab>
+            </Tabs>
+            {belowTab === BelowTabs.OWNERS && (
+              <Dropdown
+                direction="left"
+                className={styles.sortByDropdown}
+                value={orderDirection}
+                onChange={(_event, data) => {
+                  const value = data.value as OrderDirection
+                  setOrderDirection(value)
+                }}
+                options={orderDirectionOptions}
+              />
+            )}
+          </div>
+
+          {belowTab === BelowTabs.LISTINGS ? (
+            <TransactionHistory asset={item} />
```

Comment:
```suggestion
            null
```
Can we just render an empty table here? I believe that the `TransactionHistory` is going to be always rendered, underneath this new table.